plenum/test/batching_3pc/test_basic_batching.py

```diff
@@ -7,59 +7,71 @@
 from plenum.test.helper import checkReqNackWithReason, sendRandomRequests, \
     checkRejectWithReason, waitForSufficientRepliesForRequests
 from stp_core.loop.eventually import eventually
+from plenum.test.sdk.conftest import *
+from plenum.test.sdk.helper import send_random_and_check, eval_timeout
+from plenum.common.exceptions import InvalidClientRequest
 
 
-def testRequestStaticValidation(tconf, looper, txnPoolNodeSet, client,
-                                wallet1):
+def testRequestStaticValidation(tconf, looper, wallet1, testNodeClass):
     """
     Check that for requests which fail static validation, REQNACK is sent
     :return:
     """
-    reqs = [wallet1.signOp((lambda: {'something': 'nothing'})()) for _ in
-            range(tconf.Max3PCBatchSize)]
-    client.submitReqs(*reqs)
-    for node in txnPoolNodeSet:
-        looper.run(eventually(checkReqNackWithReason, client, '',
-                              node.clientstack.name, retryWait=1))
+    node = testNodeClass('Alpha')
+    req = wallet1.signOp((lambda: {'something': 'nothing'})())
```

Comment:
`wallet1` fixture (as well as Wallet and Client classes) will be deprecated. Is there a way to create a fake Request without using it?